lib/elixir/lib/kernel.ex

```diff
@@ -4020,24 +4020,18 @@ defmodule Kernel do
       is allowed and its results will be evaluated at runtime. Usually
       evaluates to the name of a module.
 
+      If the expression evaluates to a list, it will emit a warning,
+      since lists of expressions have been deprecated.
+
```

Comment:
well, it's not about promoting the feature, but "its says any expression is allows".
people will get a warning anyway. it's up to you guys